monocle/static/js/index.js

```diff
@@ -0,0 +1,3917 @@
+/* esri-leaflet - v2.0.7 - Fri Jan 06 2017 15:43:16 GMT-0800 (PST)
```

Comment:
Could you update this to the [latest](https://unpkg.com/esri-leaflet), minify it with [closure-compiler](https://developers.google.com/closure/compiler/), and rename it to *esri-leaflet.js*?